packages/plugin-core/src/commands/CreateMeetingNoteCommand.ts

```diff
@@ -0,0 +1,195 @@
+import {
+  DendronError,
+  NoteUtils,
+  SchemaCreationUtils,
+  SchemaToken,
+  SchemaUtils,
+} from "@dendronhq/common-all";
+import { vault2Path } from "@dendronhq/common-server";
+import * as fs from "fs-extra";
+import path from "path";
+import * as vscode from "vscode";
+import { PickerUtilsV2 } from "../components/lookup/utils";
+import { DENDRON_COMMANDS } from "../constants";
+import { IDendronExtension } from "../dendronExtensionInterface";
+import { ExtensionProvider } from "../ExtensionProvider";
+import { MeetingNote } from "../traits/MeetingNote";
+import { VSCodeUtils } from "../vsCodeUtils";
+import {
+  CommandOpts,
+  CreateNoteWithTraitCommand,
+} from "./CreateNoteWithTraitCommand";
+
+// internal to this class
+type ExecuteData = {
+  templateCreated: boolean;
+  schemaCreated: boolean;
+};
+
+export class CreateMeetingNoteCommand extends CreateNoteWithTraitCommand {
+  private _ext: IDendronExtension;
+  public static requireActiveWorkspace: boolean = true;
+  public static MEETING_TEMPLATE_FNAME: string = "dendron.templates.meet";
+
+  /**
+   *
+   * @param ext
+   * @param noConfirm - for testing purposes only; don't set in production code
+   */
+  constructor(ext: IDendronExtension, noConfirm?: boolean) {
+    const workspaceService = ext.workspaceService;
+
+    if (!workspaceService) {
```

Comment:
won't this always be the case if `requireActiveWorkspace = true`?

Reply:
Yes - but that depends on the logic of how the static prop `requireActiveWorkspace` works.  So if there isn't this null check / throw here, the usage of `workspaceService.config` will be a ts-error.